src/Blumchen/Subscriptions/Memoizer.cs

```diff
@@ -0,0 +1,38 @@
+namespace Blumchen.Subscriptions;
+
+internal class Memoizer<T, TU, TR>(Func<T, TU, TR> func)
```

Comment:
**SUGGESTION:** I'd vote for using ImHashMap from ImTools instead of building own code for that. It's optimised and threat-safe.

Reply:
Agreed.
closed by https://github.com/event-driven-io/Blumchen/pull/28/commits/7968c6dc2dbea0afc740c82b96c52439630f741d